### ISSUE TITLE
Crash on ios emulator

### ISSUE THREAD

**Issue by pzontrop** [epicshaggy/capacitor-native-biometric#19]
The "Usage" part of the readme is not very clear. The import statement can not correct as there is a typo in it and I do not find an interface with that name either. The only possible import I find is NativeBiometricPlugin

Yet as soon as I add the following:
```
import {NativeBiometricPlugin} from 'capacitor-native-biometric';

constructor(private biometrics: NativeBiometricPlugin) {}
```

The app crashes as soon as I enter the page.

I am using Ionic 5 and Capacitor 3. I ran the following command to get started
`npm i capacitor-native-biometric --save`
and I added the "NSFaceIDUsageDescription" to the Info.plist

Does anyone know what I did wrong?

**Comment by epicshaggy** [comment]
Do not inject NativeBiometric.  
After importing the package you can directly reference it like in the "Usage" part.

For Example
```ts
import {NativeBiometricPlugin} from 'capacitor-native-biometric';

constructor() {}

ngOnInit() {
    this.authenticate()
}

authenticate() {
     NativeBiometric.verifyIdentity({
          reason: "For easy log in",
          title: "Log in",
          subtitle: "Maybe add subtitle here?",
          description: "Maybe a description too?",
        }).then(
          () => {
            // Authentication successful
          },
          (error) => {
            // Failed to authenticate
          }
        );
}
```

**Comment by pzontrop** [comment]
That makes sense, thanks a lot for the help! I also saw the typo in the readme was fixed so I used that version as the one in your comment where you import the Interface. So:
`import {NativeBiometric} from 'capacitor-native-biometric';`
instead of
`import {NativeBiometricPlugin} from 'capacitor-native-biometric';`

The app stopped crashing and everything seems to work (sadly it seems biometrics is not available on emulators so I couldn't fully test yet)

thanks you so much!
